### PR TITLE
Create Blog “test-for-cms-v2-training”

### PR DESCRIPTION
**Pull request by sdcgsolutionslab**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/test-for-cms-v2-training.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: Test for CMS v2 training
+date: 2021-05-07T08:41:31.706Z
+author: Denis Choukroun
+authorimage: /img/Avatar1.svg
+tags:
+  - hpedev
+---
+The HPE DEV team is excited to present its revamped HPE Developer Community web portal. Featuring a fresh look and feel with easier navigation, it provides the resources you need to design and build software experiences that harness the most value from your data. With a new backend system and new design, it’s easier to use and contribute to. In addition, we’re delighted to be able to share our new Twitter handle with you, @HPE_DEV, a simplification many of you have requested.
+
+## A rich set of tools
```
Comment (Didier-Lalli) [inline content/blog/test-for-cms-v2-training.md:11]: Title should be bigger

File: content/blog/test-for-cms-v2-training.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+A key area of the portal is our Platforms section. Here, you can find APIs, GitHub repositories, and many of the other resources we make available for developers, designers, data scientists, and architects. 
+
+We host numerous platforms here, including:
```
Comment (Didier-Lalli) [inline content/blog/test-for-cms-v2-training.md:15]: Platform with no s

File: content/blog/test-for-cms-v2-training.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+* HPE Ezmeral Container Platform, 
+* HPE Ezmeral Data Fabric, 
+* HPE GreenLake
```
Comment (Didier-Lalli) [inline content/blog/test-for-cms-v2-training.md:19]: GreenLake should have a URL

**Review by Didier-Lalli — CHANGES_REQUESTED** [review]

File: content/blog/test-for-cms-v2-training.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+author: Denis Choukroun
+authorimage: /img/Avatar1.svg
+tags:
+  - hpedev
+---
+The HPE DEV team is excited to present its revamped HPE Developer Community web portal. Featuring a fresh look and feel with easier navigation, it provides the resources you need to design and build software experiences that harness the most value from your data. With a new backend system and new design, it’s easier to use and contribute to. In addition, we’re delighted to be able to share our new Twitter handle with you, @HPE_DEV, a simplification many of you have requested.
+
+## A rich set of tools
+
+A key area of the portal is our Platforms section. Here, you can find APIs, GitHub repositories, and many of the other resources we make available for developers, designers, data scientists, and architects.
```
Comment (Didier-Lalli) [inline content/blog/test-for-cms-v2-training.md:13]: This is a test

File: content/blog/test-for-cms-v2-training.md (hunk 1 above)
Comment (Didier-Lalli) [inline content/blog/test-for-cms-v2-training.md:11]: Change titke

**Review by Didier-Lalli — CHANGES_REQUESTED** [review]

File: content/blog/test-for-cms-v2-training.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+title: Test for CMS v2 training
+date: 2021-05-07T08:41:31.706Z
+author: Denis Choukroun
+authorimage: /img/Avatar1.svg
+tags:
+  - hpedev
+---
+The HPE DEV team is excited to present its revamped HPE Developer Community web portal. Featuring a fresh look and feel with easier navigation, it provides the resources you need to design and build software experiences that harness the most value from your data. With a new backend system and new design, it’s easier to use and contribute to. In addition, we’re delighted to be able to share our new Twitter handle with you, @HPE_DEV, a simplification many of you have requested.
```
Comment (choukden) [inline content/blog/test-for-cms-v2-training.md:9]: This is nice section

File: content/blog/test-for-cms-v2-training.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+## A rich set of tools
+
+A key area of the portal is our Platforms section. Here, you can find APIs, GitHub repositories, and many of the other resources we make available for developers, designers, data scientists, and architects. 
+
+We host numerous platforms here, including:
+
+* HPE Ezmeral Container Platform, 
+* HPE Ezmeral Data Fabric,
```
Comment (choukden) [inline content/blog/test-for-cms-v2-training.md:18]: URL to Ezmeral DF

**Review by choukden — CHANGES_REQUESTED** [review]

**Review by Didier-Lalli — APPROVED** [review]

**Review by choukden — APPROVED** [review]

**Pull request closed without merging**